### PR TITLE
Switching to external urls for WFS and WMS layer creation.

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java [hunk 1]
```diff
@@ -38,6 +38,8 @@
 	private static final Logger log = LoggerFactory.getLogger(GeoserverUtil.class);
 
 	private static final String geoserverEndpoint;
+	private static final String geoserverExternalEndpoint;
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java]: With this change in language for the external endpoint, you may want to refactor the variable for `geoserverEndpoint` to reflect that it's the internal endpoint.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java [hunk 2]
```diff
@@ -87,18 +90,32 @@
 //		return tmpFile;
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java]: Remove commented code

Comment (cschroed-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java]: Ah I didn't notice that. Thanks for pointing that out. I will remove the commented-out codez that pre-date this PR.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java (hunk 1 above)
Comment (cschroed-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java]: Good idear.

Comment (mhines-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/util/GeoserverUtil.java]: :+1:

**Comment by cschroed-usgs** [comment]
Thanks Ivan. I pushed a commit addressing the issues. Merging!

**Pull request merged**